financial-connections/src/main/java/com/stripe/android/financialconnections/model/SynchronizeSessionResponse.kt

```diff
@@ -21,6 +23,19 @@ internal data class TextUpdate(
     val consent: ConsentPane? = null
 ) : Parcelable
 
+@Serializable
+@Parcelize
+internal data class VisualUpdate(
+    // Indicates whether the logo should be removed from most panes
+    @SerialName("reduced_branding")
+    val reducedBranding: Boolean,
+
+    // If this field is present, it will be used in the combined logo.
+    // If it is not present, the combined logo will not include a merchant logo.
+    @SerialName("merchant_logo")
+    val merchantLogo: String?
```

Comment:
Do you need to add an `= null` here?

Reply:
good catch - https://github.com/stripe/stripe-android/pull/5949/commits/180ea2d7a122ab85c488380d4593afe2a5b6bc6f